src/scripts/install-sdk.sh

```diff
@@ -5,11 +5,21 @@ INSTALL_LOCATION=$(eval "echo $ORB_EVAL_INSTALL_LOCATION")
 if [ ! -d "$INSTALL_LOCATION/flutter" ]; then
   mkdir -p "$INSTALL_LOCATION"
   if [ "$(uname)" == 'Darwin' ]; then
-    curl -o flutter_sdk.zip https://storage.googleapis.com/flutter_infra_release/releases/stable/macos/flutter_macos_$ORB_VAL_FLUTTER_SDK_VERSION-stable.zip
+    if [ "$(uname -m)" == "arm64" ]; then
```

Comment:
i'm wondering if it might be better to just use a function to replace all the if/else/elif.

maybe something like this would be easier to read, with the install defaulting to Linux/amd64

function install_flutter() {
  local machine=${1:-"Linux"}
  local arch=${2:-"amd64"}
  [[ $machine == "Darwin" ]] && uname=macos || uname=linux
  [[ $arch == "arm64" ]] && version="${uname}_arm64" || version="${uname}"
  [[ $uname == "linux" ]] && suffix="tar.xz" || suffix="zip"

  baseurl="https://storage.googleapis.com/flutter_infra_release/releases/stable"

  [[ $arch == "arm64" ]] && version="flutter_${uname}_arm64" || version="flutter_${uname}"

  fullurl="$baseurl/$uname/${version}_$ORB_VAL_FLUTTER_SK_VERSION-stable.${suffix}"

  curl -o "flutter_sdk.${suffix}" "$fullurl"
}

install_flutter "$(uname)" "$(uname -a)"
# mac arm64
# https://storage.googleapis.com/flutter_infra_release/releases/stable/macos/flutter_macos_-stable.zip

install_flutter
# https://storage.googleapis.com/flutter_infra_release/releases/stable/linux/flutter_linux_-stable.tar.xz